src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -181,7 +181,9 @@ static void print_help_messages() {
     message_list.emplace_back("OV_GPU_DisableRuntimeSkipReorder", "Disable runtime skip reorder.");
     message_list.emplace_back("OV_GPU_DisablePrimitiveFusing", "Disable primitive fusing");
     message_list.emplace_back("OV_GPU_DisableFakeAlignment", "Disable fake alignment");
-    message_list.emplace_back("OV_GPU_EnableDynamicQuantize", "Enable Dynamic quantization for fully connected primitive");
+    message_list.emplace_back("OV_GPU_EnableDynamicQuantize", "Enable Dynamic quantization for Fully connected primitive");
+    message_list.emplace_back("OV_GPU_DynamicQuantizeLayers", "Enable Dynamic quantization for specified Fully connected layers only, separated by space."
```

Comment:
sorry but I think we need to apply either of two things
* Generalize this option to cover onednn dynamic quantization scenario
* Change the name to clarify that it is not applied to onednn case. Maybe `DynamicQuantizeLayersWithoutOnednn`?

Reply:
Applied